### PR TITLE
Added cico related files and README file for QA scripts

### PR DESCRIPTION
**Pull request by rajusem**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,28 @@
+
+DOCKERFILE := Dockerfile
+REGISTRY := quay.io
+REPOSITORY := openshiftio/osa-api-server
```
Comment (rootAvish) [inline Makefile]: Is this repository already created on quay.io?

File: cico_run_pylint.sh [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+check_python_version() {
+    python3 tools/check_python_version.py 3 6
+}
+
+# this script is copied by CI, we don't need it
+rm -f env-toolkit
+
+prep
+check_python_version
+./qa/detect-common-errors.sh
```
Comment (rootAvish) [inline cico_run_pylint.sh]: You have not added any of these scripts.

Comment (arajkumar) [reply cico_run_pylint.sh]: He has added all these script as part of previous PR.

Comment (rajusem) [reply cico_run_pylint.sh]: yes all qa scripts are part of earlier PR - https://github.com/kubesecurity/osa-api-server/pull/8

File: Makefile (hunk 1 above)
Comment (rajusem) [reply Makefile]: I raised one PR : https://gitlab.cee.redhat.com/service/app-interface/merge_requests/3667.
Is this sufficient or anything else needs to be done?

**Pull request merged**